docs/Installing.asciidoc

```diff
@@ -31,10 +31,19 @@ repository. Installation on openSUSE is therefore pretty simple:
 
 [source,sh]
 --------------------------------------------------------------------------------
-zypper ar -f obs://devel:openQA/openSUSE_13.2 openQA
-# needed for distros that lack some packages from Tumbleweed
-zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
-zypper in openQA
+
+- 13.2
+  zypper ar -f obs://devel:openQA/openSUSE_13.2 openQA
+  zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
+
+- Leap 42.1
+  zypper ar -f obs://devel:openQA/openSUSE_Leap_42.1 openQA
+  zypper ar -f obs://devel:openQA:42/openSUSE_Leap_42.1 openQA-perl-modules
```

Comment:
Paging @lnussel - am I mistaken, or did we make the devel:openQA:42 repo obsolete by putting the required modules in Leap:42.1?